kratos/modeler/surrogate_boundary_modeler.h

```diff
@@ -0,0 +1,281 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes Danes
```

Comment:
(super minor). You're mixing tabs and spaces in the header. In other words, we align the B of "BSD ..." and the K of "Kratos default ..." with the name ("A" in Ariadna).